File: problems/05-merge-and-summarize/starters/starter.py
# Problem 5: Merge and Summarize
# Merge stations.csv and observations.csv, aggregate stats
# Save output to: problems/05-merge-and-summarize/output.csv

import csv

stations = {}
with open('problems/05-merge-and-summarize/input/stations.csv', 'r') as f:
    reader = csv.DictReader(f)
    for row in reader:
        stations[row['station_id']] = row

observations = []
with open('problems/05-merge-and-summarize/input/observations.csv', 'r') as f:
    reader = csv.DictReader(f)
    for row in reader:
        observations.append(row)

# YOUR CODE HERE

File: problems/06-moving-average/starters/starter.R
# Problem 6: Moving Average
# Compute 7-day centered moving average for STN_C
# Save output to: problems/06-moving-average/output.csv

data <- read.csv('problems/05-merge-and-summarize/input/observations.csv')
data <- data[data$station_id == 'STN_C', ]
data <- data[order(data$date), ]

# YOUR CODE HERE
